### DISCUSSION TITLE
Markor Update - v2.7.0

### DISCUSSION THREAD

**Discussion by gsantner** [gsantner/markor#1374] [General]
# See [Project blog post](https://gsantner.net/blog/2021/06/26/markor-v2.7-search-in-content-backup-restore.html?source=changelog) for details!

## Changelog
- Recursive file search with in-content search, #1337 by @adelobosko @harshad1 @gsantner
- Backup and restore settings, Format selection with radio buttons, #1244 by @harshad1 @gsantner
- Per-file font size, #1332 by @harshad1
- Markdown: Support superscript^2^ syntax, #1268 by @gsantner
- View mode: Image/PDF export whole page, add seperate screenshot option, by @gsantner
- todo.txt: Fix trailing space resulting in contexts/projects/due-dates to be entered twice, #1282 by @harshad1
- Markdown: Enable GitLab extension, display video links as html5-video, #1280 by @gsantner
- Markdown: Enable typographic transformation, #1277 by @gsantner
- todo.txt: Fix tags dialog not shows up onLongClick, #1292 by @gsantner
- Editor: Allow top-menu back button also when the file is empty, #1290 by @harshad1
- ShareInto: Improve automatic link reformatting, #1275 by @harshad1
- Search dialogs: Hide search input field when if there is no data, #1298 by @harshad1
- Decrease scrollbar width for better usability, #1306 by @harshad1
- todo.txt: Settings option for always-visible @contexts & +projects, #1305 by @harshad1 @gsantner
- TextActions: Improve cursor placement at Regex replace, #1310 by @harshad1
- File manager: Move/Copy file improvements, add Yes/No/All overwrite options, #1281 by @harshad1
- Fix App might crash on toolbar-click for TOC, #1336 by @adelobosko
- Fix Android 4.4 crash on file move/copy, #1333 by @harshad1
- Fix Android 4.4 crash when opening .txt file due to ZimWiki format detection, #1341 by @fredericjacob
- Markdown: Don't match extra spaces at ordered-list regex, #1367 by @harshad1
- File Manager: Fix MB being displayed as GB at description (SI 1000 unit), #1352 by @gsantner
- File Manager: Duplicate file / allow copy into same folder, #1345 by @harshad1- Filemanager formatter: fix MB being displayed as GB (SI 1000 unit), #1352 by @gsantner

![screenshot-2021-06-26__00-59-33](https://user-images.githubusercontent.com/6735650/123492886-1766b480-d61b-11eb-8998-240805082f47.png)
![screenshot-2021-06-26__01-02-16](https://user-images.githubusercontent.com/6735650/123492889-1897e180-d61b-11eb-93f0-cbdb7f0dc452.png)
![screenshot-2021-06-26__01-02-59](https://user-images.githubusercontent.com/6735650/123492891-19307800-d61b-11eb-9d06-78b8a6315016.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/gsantner/markor/releases/tag/v2.7.0'>v2.7.0</a>.</em>

**Comment by fredericjacob** [comment]
Woohoo, the new content search feature is amazing! I really like the design of the new file search dialog and how search results are presented and grouped, seems very clean and sophisticated to me.
Thank you very much @adelobosko for your hard work on this (and also @gsantner and @harshad1 for cleaning up and reviewing), this was the only bigger feature that I was still missing in Markor! :)

Great to see how steadily this wonderful open-source app grows and progresses :+1:
